summary_workflows/quantification/quantification_dockers/q2_consolidation/assessment_chart/assessment_chart.py

```diff
@@ -242,12 +242,12 @@ def print_chart(cancer_dir, summary_dir, cancer_type, classification_type):
     # change plot style
     # set plot title
 
-    plt.title("Cancer Driver Genes prediction benchmarking - " + cancer_type, fontsize=18, fontweight='bold')
+    plt.title("Quantification benchmark 2 - number of matched sites vs number of unmatched sites in challenge" + challenge_type, fontsize=18, fontweight='bold')
 
     # set plot title depending on the analysed tool
 
-    ax.set_xlabel("True Positive Rate - % driver genes correctly predicted", fontsize=12)
-    ax.set_ylabel("Precision - % true positives over total predicted", fontsize=12)
+    ax.set_xlabel("Number of matched sites", fontsize=12)
+    ax.set_ylabel("Number of unmatched sites", fontsize=12)
```

Comment:
Numbers of matched and unmatched sites are not relevant for quantification, but may be used in identification benchmarks. The only metric in this benchmark should be correlation.
However, if we want to use it in Q2 for now anyway, it would me more informative to calculate the percentage of matched sites, e.g. %matched = matched / (matched + unmatched) and display it as a barplot.